Jenkinsfile

```diff
@@ -25,7 +25,7 @@ pipeline {
         maven('maven')
         git('git')
         ant('Ant')
-        jdk('JDK11')
+        jdk('OpenJDK11')

```

Comment:
As discusssed, I'll wait for your confirmation on which test (PR branch or master) we want to make to merge this one.

Reply:
Looks like it worked. I did a test on this PR branch and the deployment succeeded (4.0.4-SNAPSHOT).